Deployment/Kubernetes/TensorRT-LLM_Autoscaling_and_Load_Balancing/chart/pvc/storageclass.yaml

```diff
@@ -0,0 +1,5 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: efs-autoscaling-sc
+provisioner: efs.csi.aws.com
```

Comment:
why are we checking in AWS specific solutions?

do we plan to provide GCloud and/or Azure options as well?

Reply:
My ideal behind this was to show "some" kind of NFS solution.
AWS just was the most easiest to implement.
I have tried making it clear in our README that this is just ONE of the ways of doing it.
My thoughts were if we just say "use NFS" people might run into issues, hence better to show one working example.
I am happy to remove this if we don't want to check-in AWS specific solution.

Reply:
then it should separate from the core charts. Putting AWS specific solutions into the core charts prevents users on other solutions from leveraging the charts completely.

Reply:
Moved all pvc charts for aws under pvc_aws folder